pkg/kube/kube_test.go

```diff
@@ -129,6 +129,39 @@ var _ = Describe("GetClusterName", func() {
 	})
 })
 
+var _ = Describe("FixInvalidClusterName", func() {
```

Comment:
There is a ginkgo extension that can help make these independent test cases: https://pkg.go.dev/github.com/onsi/ginkgo/extensions/table

Reply:
Just pushed a change to use the extension.